src/index.html

```diff
@@ -13,10 +13,20 @@
     <title>Stop watch</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/main.scss"
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div
+        class="stopwatch__minutes-hand stopwatch__minutes-hand--speed-up"
```

Comment:
Let's shorten it to to make it more readable (here and for other div's)

`  <div class="stopwatch__minutes-hand stopwatch__minutes-hand--speed-up" />`